### PR TITLE
func-e 1.1.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
```
==> Testing func-e
==> /opt/homebrew/Cellar/func-e/1.1.4/bin/func-e run --version
dyld: Symbol not found: __ZNKSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE3strEv
  Referenced from: /private/tmp/func-e-test-20230105-69203-1bzeb1u/.func-e/versions/1.24.1/bin/envoy (which was built for Mac OS X 13.0)
  Expected in: /usr/lib/libc++.1.dylib

Error: func-e: failed
```

**Comment by cho-m** [comment]
```console
❯ c++filt __ZNKSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE3strEv
std::__1::basic_stringbuf<char, std::__1::char_traits<char>, std::__1::allocator<char> >::str() const
```

Somewhat similar symbol to `creduce` failure which I never figured out reason for:
https://github.com/Homebrew/homebrew-core/blob/c8fe0ad9d97930b5c04fae66963435ca8f47bd87/Formula/creduce.rb#L96-L99

**Comment by codefromthecrypt** [comment]
hmm the version of envoy func-e uses on mac is from homebrew ;)

**Comment by codefromthecrypt** [comment]
func-e can't run on an operating system that its archive doesn't verify. https://github.com/tetratelabs/archive-envoy/pull/42 dropped MacOS 11, so is that it?

**Comment by codefromthecrypt** [comment]
FWIW earlier versions of envoy work, I forget which started forking the road on OS/x support, but also I'm not involved in envoy stuff for a year now.

**Comment by chenrui333** [comment]
> hmm the version of envoy func-e uses on mac is from homebrew ;)

envoy 1.24.1

**Comment by chenrui333** [comment]
> func-e can't run on an operating system that its archive doesn't verify. [tetratelabs/archive-envoy#42](https://github.com/tetratelabs/archive-envoy/pull/42) dropped MacOS 11, so is that it?

hmm, does that mean, func-e cannot run the homebrew copy of envoy (which has macos-11 support)?

**Comment by chenrui333** [comment]
> ```
> ❯ c++filt __ZNKSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE3strEv
> std::__1::basic_stringbuf<char, std::__1::char_traits<char>, std::__1::allocator<char> >::str() const
> ```
> 
> Somewhat similar symbol to `creduce` failure which I never figured out reason for:
> 
> https://github.com/Homebrew/homebrew-core/blob/c8fe0ad9d97930b5c04fae66963435ca8f47bd87/Formula/creduce.rb#L96-L99

looks very similar.

**Comment by codefromthecrypt** [comment]
TL;DR; easiest way out is to drop MacOS 11 support for func-e.

Basically, binaries built after a certain version no longer work on multiple versions of OS/x, notably 11. archive-envoy only archives one version per os/arch. So, if an implicit envoy version is used in tests, then it should not try to validate on MacOS 11 anymore, even if func-e can run that. I don't expect anyone to change archive-envoy to support 11 either.

**Comment by chenrui333** [comment]
Sounds good. 👍

**Review by p-linnane — APPROVED** [review]
